superset/commands/dashboard/update.py

```diff
@@ -202,3 +203,29 @@ def run(self) -> Model:
         )
 
         return configuration
+
+
+class UpdateDashboardColorsConfigCommand(UpdateDashboardCommand):
+    @transaction(
```

Comment:
Isn't this decorator supposed to be used on the `def run`? hence no need to disable the pylint rule about transaction

Reply:
You are right that this should be decorating the `run`. Just moved it. As for disabling the rule, that's to commit the modified date back to its original state when the `mark_updated` parameter is set to `False`. In all other cases, it should commit normally using the `@transaction`.